demo/src/style/app.scss

```diff
@@ -178,12 +178,12 @@ div.component-demo {
   h2 {
     margin-bottom: 1rem;
 
-    .plunker {
+    .plunker, .stackblitz {
       opacity: 0.3;
       transition: opacity .5s;
     }
     &:hover {
-      .plunker {
+      .plunker, .stackblitz {
```

Comment:
I guess you should add `text-decoration: none` here, to remove the a:hover blue line glitch between two icons:

<img width="227" alt="screen shot 2018-03-09 at 09 38 35" src="https://user-images.githubusercontent.com/8074436/37198477-b1e838ec-237e-11e8-90f9-67adbebd12e0.png">